### PR TITLE
Nit: Change constant to ALL_CAPS

### PR DESCRIPTION
**Pull request by johhat**
True to standard python conventions, the constant is here changed to upper case letters with words separated by underscore.

### CODE REVIEW

File: src/initial.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import pandas as pd
 from matplotlib import pyplot as plt
 
-num_measurements = 25
+NUM_MEASUREMENTS = 25
```
Comment (sunnivin) [inline src/initial.py:4]: WTF!

File: src/initial.py [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 # read data from file
-data = pd.read_csv('data/temperatures.csv', nrows=num_measurements)
+data = pd.read_csv('data/temperatures.csv', nrows=NUM_MEASUREMENTS)
 temperatures = data['Air temperature (degC)']
 
 # compute statistics
-mean = sum(temperatures)/num_measurements
+mean = sum(temperatures)/NUM_MEASUREMENTS
```
Comment (sunnivin) [inline src/initial.py:11]: Yeah, this was a good idea!

**Review by sunnivin — APPROVED** [review]
LGTM

**Comment by johhat** [comment]
Thank you. Very insightful comments.

File: src/initial.py [hunk 3]
```diff
@@ -1,14 +1,14 @@
 import pandas as pd
 from matplotlib import pyplot as plt
 
-num_measurements = 25
+NUM_MEASUREMENTS = 25
 
 # read data from file
-data = pd.read_csv('data/temperatures.csv', nrows=num_measurements)
```
Comment (sunnivin) [inline src/initial.py:7]: what about changing blabla

**Review by sunnivin — COMMENTED** [review]
LGTM

**Pull request merged**